### ISSUE TITLE
//pkg/kubelet/pleg:go_default_test is flaky

### ISSUE THREAD

**Issue by fejta** [kubernetes/kubernetes#76989]
@kubernetes/sig-node-bugs 

```
FAIL: //pkg/kubelet/pleg:go_default_test (see /usr/local/google/home/fejta/.cache/bazel/_bazel_fejta/631ceaac9e4091781154c1030ef08d3e/execroot/__main__/bazel-out/k8-fastbuild/testlogs/pkg/kubelet/pleg/go_default_test/test_attempts/attempt_1.log)

FLAKY: //pkg/kubelet/pleg:go_default_test (Summary)
      /usr/local/google/home/fejta/.cache/bazel/_bazel_fejta/631ceaac9e4091781154c1030ef08d3e/execroot/__main__/bazel-out/k8-fastbuild/testlogs/pkg/kubelet/pleg/go_default_test/test.log
      /usr/local/google/home/fejta/.cache/bazel/_bazel_fejta/631ceaac9e4091781154c1030ef08d3e/execroot/__main__/bazel-out/k8-fastbuild/testlogs/pkg/kubelet/pleg/go_default_test/test_attempts/attempt_1.log
INFO: From Testing //pkg/kubelet/pleg:go_default_test:
==================== Test output for //pkg/kubelet/pleg:go_default_test:
E0424 06:42:20.674764      18 generic.go:274] event channel is full, discard this relist() cycle event
--- FAIL: TestEventChannelFull (0.00s)
    generic_test.go:97: Actual events differ from the expected; diff:
         [{"ID":"1234","Type":"ContainerRemoved","Data":"c1"},{"ID":"1234","Type":"ContainerDied","Data":"c2"},{"ID":"
        
        A: 1234","Type":"ContainerStarted","Data":"c3"},{"ID":"4567","Type":"ContainerRemoved","Data":"c1"}]
        
        B: 4567","Type":"ContainerRemoved","Data":"c1"},{"ID":"4567","Type":"ContainerStarted","Data":"c4"}]
        
FAIL
```

**Comment by fejta** [comment]
Repro:
```shell
bazel test --config=unit //pkg/kubelet/pleg:go_default_test --runs_per_test=30
```

**Comment by danielqsj** [comment]
@changyaowei: this test case was introduced by https://github.com/kubernetes/kubernetes/commit/b52afc350ff93cf2b26831ef4b65e2610600d782#diff-49a1185cd859eb6ef8ffc6cf807a7feb .  Could you verify it ?
cc @resouer

**Comment by mattjmcnaughton** [comment]
I believe this is fixed by https://github.com/kubernetes/kubernetes/pull/76603 - which is waiting on a merge. I'll re-ping the assigned reviewer :)

**Comment by changyaowei** [comment]
I already fixed it in #76603 , it waiting to be merged.
